rocAL/rocAL/source/meta_data/meta_data_reader_factory.cpp

```diff
@@ -38,24 +38,26 @@ THE SOFTWARE.
 #include "video_label_reader.h"
 #include "mxnet_meta_data_reader.h"
 
-std::shared_ptr<MetaDataReader> create_meta_data_reader(const MetaDataConfig& config) {
+std::shared_ptr<MetaDataReader> create_meta_data_reader(const MetaDataConfig& config, pMetaDataBatch& meta_data_batch) {
     switch(config.reader_type()) {
         case MetaDataReaderType::FOLDER_BASED_LABEL_READER:
         {
             if(config.type() != MetaDataType::Label)
                 THROW("FOLDER_BASED_LABEL_READER can only be used to load labels")
             auto ret = std::make_shared<LabelReaderFolders>();
```

Comment:
rename ret to meta_data_reader for better readability

Reply:
Done